docs/concepts/configuration/assign-pod-node.md

```diff
@@ -190,13 +190,7 @@ flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
 
 As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
 
-In principle, the `topologyKey` can be any legal label value. However,
-for performance reasons, only a limited set of topology keys are allowed.
-By default the allowed topology keys are:
-
-* `kubernetes.io/hostname`
-* `failure-domain.beta.kubernetes.io/zone`
-* `failure-domain.beta.kubernetes.io/region`
+The `topologyKey` can be any legal label value.
```

Comment:
Thanks @gyliu513 @kevin-wangzefeng , may I say  like this:
topologyKey can be set to any any legal label value.
If there is no field `topologyKey` in the pod's spec and the admission controller "LimitPodHardAntiAffinityTopology" isn't enabled, it means "all topologies", and its value comes from kubeletapis.DefaultFailureDomains.